scripts/install_offline_fs_kbc_files.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Alibaba Cloud
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+script_dir="$(dirname $(readlink -f $0))"
+resource_json_name="aa-offline_fs_kbc-resources.json"
+test_resource_record="${script_dir}/../test_data/offline-fs-kbc/${resource_json_name}"
+target_file_path="/etc/${resource_json_name}"
+
+if [ $1 == "install" ]; then
+    cp "${test_resource_record}" "${target_file_path}"
```

Comment:
I'd consider:

- Removing one of the equals signs (as it's not required).
- Using `sudo` to allow this to run for non-`root` users who wish to run the tests.
- Being explicit about the perms and ownership

... resulting in something like:

```suggestion
if [ "${1:-}" = "install" ]; then
    sudo install --owner=root --group=root --mode=0640 "${test_resource_record}" "${target_file_path}"
```